src/language_features/lean.rs

```diff
@@ -77,3 +77,76 @@ pub fn plain_goal(meta: EditorMeta, params: EditorPlainGoalParams, ctx: &mut Con
         },
     );
 }
+
+#[derive(Debug, Clone)]
+pub struct EditorPlainTermGoalParams {
+    pub position: KakounePosition,
+    pub buffer: String,
+}
```

Comment:
I think we should reuse `EditorPlainGoalParams` here,
since the two are unlikely to deviate.